code.py

```diff
@@ -17,9 +17,23 @@ def __init__(self, numBots, lines, pose):
 		for index, val in enumerate(self.lines):
 			self.group[index % self.bots] = np.append(self.group[index % self.bots], np.array([val]), axis = 0)
 		print(self.group)
-
+		
+		#length of the lines
+		self.length = []
+		for l in lines:
+			x1 = l[0][0]
+			y1 = l[0][1]
+			x2 = l[1][0]
+			y2 = l[1][1]
+			
+			#print (x1,y1,x2,y2)
+			dist = math.sqrt((x2 - x1)**2 + (y2 - y1)**2)
```

Comment:
This is perfect, but np.linalg.norm might be easy to use

Reply:
Oh okay. I'll change it in the next pull req